tybalt/models.py

```diff
@@ -309,6 +299,14 @@ def train_adage(self, train_df, test_df):
                                         validation_data=(np.array(test_df),
                                                          np.array(test_df)))
 
+    def connect_layers(self):
```

Comment:
Does this need to be explicitly called? Could this function be integrated into the initialized_model function?